misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/ResultFeatureAssertionsSpec.kt

```diff
@@ -16,6 +16,8 @@ abstract class ResultFeatureAssertionsSpec(
     isSuccess: Fun1<Result<Int>, Expect<Int>.() -> Unit>,
     isSuccessNullableFeature: Feature0<Result<Int?>, Int?>,
     isSuccessNullable: Fun1<Result<Int?>, Expect<Int?>.() -> Unit>,
+    isFailureFeature: Feature0<Result<Int>, IllegalArgumentException>,
+    isFailure: Feature1<Result<Int>, Expect<IllegalArgumentException>.() -> Unit, IllegalArgumentException>,
     describePrefix: String = "[Atrium] "
 ) : Spek({
```

Comment:
please add isFailureFeature and isFailure also to the SubjectLessSpec (line 25) and isFailure to the AssertionCreatorSpec (line 35)

Reply:
For the `AssertionCreatorSpec`, instead of adding `isFailure` to the one on line 35, I created a new one where the subject is a `Result.failure(...)`.
The reason for this is that: because the subject in the `AssertionCreatorSpec` on line 35 is a `Result.success()`, `isFailure` fails before getting to the first assertionCreator-lambda.